### PR TITLE
Configure the provisioner node with a IPv4 address

### PR DESCRIPTION
**Pull request by jorti**
The default provisioningNetworkCIDR is 172.22.0.0/24. Configure the
first IP address in the provisioner node so it can communicate with the
bootstrap node.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@rolfedh, will you PTAL?

File: modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc [hunk 1]
```diff
@@ -134,7 +134,7 @@ $ sudo nohup bash -c "
     nmcli con delete \"$PROV_CONN\"
     nmcli connection add ifname provisioning type bridge con-name provisioning
     nmcli con add type bridge-slave ifname \"$PROV_CONN\" master provisioning
-    nmcli connection modify provisioning ipv6.addresses fd00:1101::1/64 ipv6.method manual
+    nmcli connection modify provisioning ipv4.addresses 172.22.0.1/24 ipv4.method manual ipv6.addresses fd00:1101::1/64 ipv6.method manual
```
Comment (rolfedh) [inline modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc:137]:
```suggestion
    nmcli connection modify provisioning ipv4.addresses 172.22.0.1/24 ipv4.method \
  manual ipv6.addresses fd00:1101::1/64 ipv6.method manual
```
You might want to insert a backslash and line break as recommended by the "[Formatting](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting)" section in the Supplemental Style Guide:
"For long lines of code that you want to break up among multiple lines, use a backslash to show the line break. For example:

$ oc get endpoints --all-namespaces --template \
    '{{ range .items }}{{ .metadata.namespace }}:{{ .metadata.name }} \
    {{ range .subsets }}{{ range .addresses }}{{ .ip }} \
    {{ end }}{{ end }}{{ "\n" }}{{ end }}' | awk '/ 172\.30\./ { print $1 }'

Comment (jorti) [reply modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc]: Done.

**Review by rolfedh — APPROVED** [review]

**Comment by jorti** [comment]
I see that `modules/ipi-install-configuring-networking.adoc` has already added a point to configure an optional IPv4 address in the provisioning network. Also, the `provisioningNetworkCIDR` value in `install-config.yaml` is now generic, so I don't think this commit is needed anymore.

I'm closing it.
Thank you.

**Pull request closed without merging**